platform_versioned_docs/version-24.1/seqerakit/installation.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: "Installation"
+description: "seqerakit installation options"
+date: "21 Oct 2024"
+tags: [seqerakit, cli, automation, installation]
+---
+
+
+`seqerakit` is a Python wrapper that sets [Platform CLI](../cli/overview.mdx) command options using YAML configuration files. Individual seqerakit commands and configuration parameters can be chained together to automate the end-to-end creation of all Seqera Platform entities.
+
+As an extension of the Platform CLI, seqerakit enables:
+
+- **Infrastructure as code**: Users manage and provision their infrastructure from the command line.
+- **Simple configuration**: All Platform CLI command-line options can be defined in simple YAML format.
+- **Automation**: End-to-end creation of Seqera entities, from adding an organization to launching pipelines.
+
+### Installation 
+
+`seqerakit` has three dependencies:
+
+1. [Seqera Platform CLI (`>=0.9.2`)](https://github.com/seqeralabs/tower-cli/releases)
+2. [Python (`>=3.8`)](https://www.python.org/downloads/)
+3. [PyYAML](https://pypi.org/project/PyYAML/)
+
+#### Pip
+
+If you already have [Seqera Platform CLI](../cli/installation.mdx) and Python installed on your system, install `seqerakit` directly from [PyPI](https://pypi.org/project/seqerakit/):
+
+```console
+pip install seqerakit
+```
+
+Overwrite an existing installation to use the latest version:
+
+```console
+pip install --upgrade --force-reinstall seqerakit
+```
+
+#### Conda
+
+To install `seqerakit` and its dependencies via Conda, first configure the correct channels:
+
+```console
+conda config --add channels bioconda
+conda config --add channels conda-forge
+conda config --set channel_priority strict
+```
+
+Then create a conda environment with `seqerakit` installed:
+
+```console
+conda env create -n seqerakit seqerakit
+conda activate seqerakit
+```
+
+#### Local development installation
+
+Install the development branch of `seqerakit` on your local machine to test the latest features and updates:
```

Comment:
I would argue the opposite here. In my mind, this is specifically referring to the command-line tool rather than the product, and it should be `seqerakit`.